packages/libs/components/src/map/DonutMarker.tsx

```diff
@@ -215,11 +215,18 @@ function donutMarkerSVGIcon(props: DonutMarkerStandaloneProps): {
 } {
   const scale = props.markerScale ?? MarkerScaleDefault;
   const size = 40 * scale;
+  // set outter white circle size to describe white boundary
+  const outerWhiteCircleSize = size / 2 + size / 16;
```

Comment:
Can we call this `backgroundWhiteCircleRadius`? The 'background' because we see it's outer and inner parts since it's behind the data donut, and 'radius' because it looks like we usually use 'size' as a diameter but below in line 252 we're using it as a radius.

Reply:
@asizemore Thank you for your reviews, tests, and suggestions! 👍 I like the word so will change it accordingly.